### PR TITLE
Allow scripted thread plans to modify the thread stop description when

### PR DESCRIPTION
**Pull request by jimingham**
they are completed.

(cherry picked from commit c2be702104284cb3facf31124494b9a400296179)

### CODE REVIEW

**Comment by jimingham** [comment]
@swift-ci please test

**Comment by jimingham** [comment]
I needed to update the static bindings since they don't get built by this bot.

**Comment by jimingham** [comment]
@swift-ci please test

**Comment by jimingham** [comment]
Huh, that windows test run has been pending for quite some time now.  Let's try again...

**Comment by jimingham** [comment]
@swift-ci please test windows

File: lldb/include/lldb/Target/ThreadPlanPython.h [hunk 1]
```diff
@@ -64,6 +67,7 @@ class ThreadPlanPython : public ThreadPlan {
   StructuredDataImpl m_args_data;
   std::string m_error_str;
   StructuredData::ObjectSP m_implementation_sp;
+  StreamString m_stop_description; // Cache the stop description here
```
Comment (adrian-prantl) [inline lldb/include/lldb/Target/ThreadPlanPython.h:70]:
```
/// Cache the stop description here.
StreamString m_stop_description;
```
would be doxygen-friendly.

**Review by adrian-prantl — APPROVED** [review]

**Pull request merged**